packages/firestore/src/remote/remote_store.ts

```diff
@@ -192,7 +192,7 @@ export class RemoteStore {
       this.watchStreamFailures++;
       if (this.watchStreamFailures >= ONLINE_ATTEMPTS_BEFORE_FAILURE) {
         if (this.shouldWarnOffline) {
-          log.debug(LOG_TAG, 'Could not reach Firestore backend.');
+          log.error(LOG_TAG, 'Could not reach Firestore backend.');
```

Comment:
Pull #376 is removing tags from `log.error` calls. Let's please not add another one :-).